lib/config/go_routes/go_router.dart

```diff
@@ -208,6 +209,25 @@ abstract class AppRoutes {
                 ),
               ],
             ),
+            GoRoute(
+              path: 'profileinfo/:roomid/:userid',
+              pageBuilder: (context, state) {
+                final roomId = state.pathParameters['roomid'];
+                final userId = state.pathParameters['userid'];
+
+                if (roomId == null || userId == null) {
+                  return defaultPageBuilder(context, const ErrorPage());
+                }
+
+                return defaultPageBuilder(
```

Comment:
use `CupertinoPage` to enable swipe to close IOS in mobile

Reply:
i think you should use `Navigator.push` because if not you swipe back, it will back to chat screen

Reply:
I thought that's what it should do: on swipe back or on tap on back button go back to group informations screen.

https://github.com/linagora/twake-on-matrix/pull/1596#issuecomment-2018100783

https://github.com/linagora/twake-on-matrix/assets/31937920/b3d51d48-29b4-41c8-9278-3750a1234cd5

Reply:
https://github.com/linagora/twake-on-matrix/pull/1596#issuecomment-2018295049

Reply:
okay i pull again and test for you in ios

Reply:
FYI I just did a quickfix to close the bottom sheet after clicking on the button

Reply:
i send the video in the chat. There is another reason to use Navigator, in the video you can that the animation for swipe of GoRouter is no smooth

Reply:
And i found that they have not fix that.

Reply:
So i think if you can change to use navigator, it's better and synchonize with the screen inside chat

Reply:
https://github.com/linagora/twake-on-matrix/assets/31937920/9d0c5727-e0cb-46b1-9a31-19f26d37fbcd

Done

Reply:
https://github.com/linagora/twake-on-matrix/assets/43041967/432d1430-9e25-4935-b6cd-ffce6b5cd5ff

Reply:
okey, it's good, dont forget to remove the unnecessary code